endpoints/users_id_token.py

```diff
@@ -316,6 +316,25 @@ def _set_oauth_user_vars(token_info, audiences, allowed_client_ids, scopes,
 # pylint: enable=unused-argument
 
 
+def _process_scopes(scopes):
```

Comment:
Maybe document what `scopes` is?
I believe `scopes` should be a list of string, and each string is a space-separated list of scope.
Is this correct?

Reply:
Oh yeah, whoops!